code/API_definitions/qod-api.yaml

```diff
@@ -222,7 +222,46 @@ paths:
         - oAuth2ClientCredentials: []
         - threeLegged:
             - "qod-sessions-write"
-
+    get:
+      tags:
+        - QoS sessions
+      summary: Get all session information
+      description: Get information about all active sessions that were created by the client
+      operationId: getSessions
+      parameters:
+        - name: perPage
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/PerPage"
+        - name: page
+          in: query
+          required: false
+          schema:
+            $ref: "#/components/schemas/Page"
```

Comment:
I would suggest to call it 'pageNumber', if your intention is to highlight which page should be shown.